### DISCUSSION TITLE
blog/better-type-safety-for-resolvers-with-graphql-codegen

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1355] [Docs Discussions]
# blog/better-type-safety-for-resolvers-with-graphql-codegen

Type-check your resolvers' implementation in order to find issues in build-time. Simply integrate with your existing TypeScript data models and context types.

https://the-guild.dev/blog/better-type-safety-for-resolvers-with-graphql-codegen

**Comment by sahil-shubham** [comment]
The codesandbox examples aren't available because of lack of credits, cannot form them as well :/

**Comment by henningms** [comment]
You're (still?) out of credits. Maybe replace CodeSandbox examples with plain code samples..
